thirdeye/thirdeye-frontend/app/styles/main.scss

```diff
@@ -0,0 +1,72 @@
+@import "shared/variable";
+
+hr {
+  margin: 0;
+}
+
+.hide {
```

Comment:
this is a BEM anti-pattern. 
For reusable css class, let's plan on making them mixins and including them in your modifier classes

i.e 
```
.te-navbar__item--hide {
 @include hide-mixin
}
```